record_web/lib/record_web.dart

```diff
@@ -163,11 +196,17 @@ class RecordPluginWeb extends RecordPlatform {
 
   @override
   Future<Amplitude> getAmplitude() async {
-    // TODO how to check amplitude values on web?
-    return Amplitude(current: -160.0, max: -160.0);
+    try {
+      var amp = getMaxAmplitude();
+      if (maxAmplitude < amp) {
+        maxAmplitude = amp;
+      }
+      return Amplitude(current: amp, max: maxAmplitude);
+    } catch (e) {
+      return Amplitude(current: -160, max: -160);
```

Comment:
Use new const min and max

Reply:
here see the current change it is needed to return Amplitude(current: minmumAmplitude, max: maxAmplitudeRecorded);
maxAmplitudeRecorded if not changed at all it will be the minimum which logically the right value